functions/tslint.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
We could/should use [ESLint](https://eslint.org/) for the whole project. It's the defacto right now and it works with almost anything JS related.

Reply:
I think we need to use tslint with typescript in order to support type annotations.

Reply:
I think there's a TypeScript plugin for eslint, but we should investigate first to see if it's really better than TSLint. Never used either.

Reply:
We should take a look to compare both.